src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -658,5 +659,14 @@ public Party Approvals
                 return approvals;
             }
         }
+
+        public void TransferRequestRejected(UserInfo userInfo)
```

Comment:
Can we rename this to `RejectTransferRequest` - I don't like the past-tense used; it makes it sound like an event.